packages/kubiscan/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=kubiscan
+_pkgname=KubiScan
+pkgver=195.b851b37
+pkgrel=1
+pkgdesc='A tool for scanning a Kubernetes cluster for risky rbac permissions.'
+arch=('any')
+groups=('blackarch' 'blackarch-scanner')
+url='https://github.com/cyberark/KubiScan'
+license=('GPL3')
+depends=('python' 'python-virtualenv')
+makedepends=('git' 'python-pip')
+source=("git+https://github.com/cyberark/$_pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
+
+pkgver() {
+  cd $_pkgname
+
+  echo "$(git rev-list --count HEAD).$(git rev-parse --short HEAD)"
```

Comment:
Could you use the version with tags, as git tags are available.

Reply:
Will fix